src/roles/foreman/tasks/main.yaml

```
@@ -205,6 +260,16 @@
   delay: 5
   register: foreman_status
 
+- name: Enable & start recurring timers
+  when: foreman_recurring_tasks_enabled | default(true)
```

```suggestion
  when: foreman_recurring_tasks_enabled
```
You already define it in `roles/foreman/defaults/main.yaml`, so no need to duplicate the default here.